src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsperiode/StønadsperiodeValideringUtilTest.kt

```diff
@@ -384,6 +386,112 @@ internal class StønadsperiodeValideringUtilTest {
         }
     }
 
+    @Nested
+    inner class OverlappMedPeriodeSomIkkeGirRettPåStønad {
+
+        val jan = YearMonth.of(2024, 1)
+        val fom = jan.atDay(10)
+        val tom = jan.atDay(20)
+
+        @Test
+        fun `kan ha stønadsperiode før og etter periode som ikke gir rett på stønad`() {
+            val målgrupper = listOf(
+                målgruppe(
+                    type = MålgruppeType.SYKEPENGER_100_PROSENT,
+                    fom = fom,
+                    tom = tom,
+                    begrunnelse = "asd",
+                    resultat = ResultatVilkårperiode.IKKE_OPPFYLT,
+                ),
+                målgruppe(
+                    type = MålgruppeType.INGEN_MÅLGRUPPE,
+                    fom = fom,
+                    tom = tom,
+                    begrunnelse = "asd",
+                    resultat = ResultatVilkårperiode.IKKE_OPPFYLT,
+                ),
+                målgruppe(type = MålgruppeType.AAP, fom = jan.atDay(1), tom = jan.atDay(9)),
```

Comment:
Kan denne raden flyttes over målgruppe med sykepenger for å visualisere at denne er før?